### PR TITLE
[load-themed-styles] Check for document in loadStyles/registerStyles

### PR DESCRIPTION
**Pull request by bkniffler**

### CODE REVIEW

File: libraries/load-themed-styles/src/index.ts [hunk 1]
```diff
@@ -164,6 +164,9 @@ function initializeThemeState(): IThemeState {
  * @param {boolean} loadAsync When true, always load styles in async mode, irrespective of current sync mode.
  */
 export function loadStyles(styles: string | ThemableArray, loadAsync: boolean = false): void {
+  if (typeof document === 'undefined') {
```
Comment (pgonzal) [inline libraries/load-themed-styles/src/index.ts]: >if (typeof document === 'undefined') { [](start = 2, length = 38)

Could you add a code comment indicating the usage scenario where "document" might be undefined?

Your GitHub issue discussed this, but someone reading this code wouldn't have that context

**Comment by pgonzal** [comment]
Your PR build failed because you're missing a changelog

See the "**rush change**" section of the [Everyday Commands](https://rushjs.io/pages/developer/everyday_commands/) docs.

File: libraries/load-themed-styles/src/index.ts (hunk 1 above)
Comment (bkniffler) [reply libraries/load-themed-styles/src/index.ts]: Shouldn't it be trivial for anyone looking at this to know this just exits on non-DOM environments? Also this escape hatch is used further down without any comments.

Comment (pgonzal) [reply libraries/load-themed-styles/src/index.ts]: Fair enough

**Comment by pgonzal** [comment]
@bkniffler  I pushed a change log, but it seems that one of the unit tests is failing.  Could you take a look? Maybe `registerStyles()` or `loadStyles()` do some useful work even if `document` is undefined

**Comment by pgonzal** [comment]
Also we need you to sign the CLA

**Comment by pgonzal** [comment]
:shipit:

**Comment by pgonzal** [comment]
@bkniffler thanks a ton for contributing this fix!  I just published it as **load-themed-styles** 1.8.50

**Pull request merged**